### PR TITLE
[DreamBooth] Set train mode for text encoder

### PR DESCRIPTION
**Pull request by duongna21**
cc @patil-suraj

### CODE REVIEW

**Review by patil-suraj — APPROVED** [review]
Good catch, thanks a lot for the PR @duongna21 !

**Comment by Twizzes** [comment]
Hi - just curious... does this mean that we weren't truly training the text encoder before?

**Comment by duongna21** [comment]
@Twizzes Not really. It means that Dropout layers in text encoder will be turned off during training. This shouldn't make significant impact in the context of finetuning.
